functions/index.js

```diff
@@ -5,8 +5,8 @@ const admin = require('firebase-admin');
 const path = require('path');
 
 admin.initializeApp();
-
-const BUCKET_NAME = 'barnard-project-audio'
+const projectId = admin.instanceId().app.options.projectId 
```

Comment:
Looks good, but the project id is still hardcoded on the frontend, in ApiUtils.js.

Reply:
also, please add semicolons to your added lines in this file!

Reply:
Done.